### PR TITLE
Usability improvements

### PR DESCRIPTION
**Pull request by JsseL**

### CODE REVIEW

File: cmsplugin_carousel_ai/models.py [hunk 1]
```diff
@@ -21,6 +21,14 @@ class Carousel(CMSPlugin):
     def __str__(self):
         return "%s" % self.name
 
+    @property
+    def interval_in_ms(self):
```
Comment (akx) [inline cmsplugin_carousel_ai/models.py:25]: Maybe `msec_interval`?

File: cmsplugin_carousel_ai/models.py [hunk 2]
```diff
@@ -21,6 +21,14 @@ class Carousel(CMSPlugin):
...
+        """
+        Return the slide change interval time in milliseconds.
+        :retval type: Int
```
Comment (akx) [inline cmsplugin_carousel_ai/models.py]: `:rtype: int`

File: cmsplugin_carousel_ai/models.py [hunk 3]
```diff
@@ -21,6 +21,14 @@ class Carousel(CMSPlugin):
     def __str__(self):
         return "%s" % self.name
 
+    @property
+    def interval_in_ms(self):
+        """
+        Return the slide change interval time in milliseconds.
```
Comment (akx) [inline cmsplugin_carousel_ai/models.py]: It's a property, it always returns something.

> The slide change interval in milliseconds

is enough.

File: cmsplugin_carousel_ai/models.py [hunk 4]
```diff
@@ -85,3 +98,11 @@ def link(self):
     @property
     def has_link(self):
         return bool(self.linked_page_id or self.url)
+
+    @property
+    def has_call_to_action(self):
+        """
+        Return true if slide has proper information for creating
```
Comment (akx) [inline cmsplugin_carousel_ai/models.py]: Maybe

> Whether the slide has the required information for Call To Actions.
> :rtype: bool

File: cmsplugin_carousel_ai/models.py (hunk 1 above)
Comment (JsseL) [reply cmsplugin_carousel_ai/models.py:25]: I find the existing more readable.

**Pull request merged**